2021-05-18 13:02:20,557 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-05-18 13:02:20,557 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/hans/Library/Python/3.8/lib/python/site-packages/instabot/bot/bot.py
2021-05-18 13:02:43,475 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-05-18 13:02:43,730 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-05-18 13:02:43,731 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-05-18 13:02:43,731 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-05-18 13:02:43,731 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2021-05-18 13:07:44,027 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-05-18 13:07:44,029 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-05-18 13:07:44,029 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-05-18 13:07:44,029 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 10 minutes.
